crates/eframe/src/native/wgpu_integration.rs

```diff
@@ -954,8 +949,6 @@ fn render_immediate_viewport(
         input
     };
 
-    let egui_ctx = shared.borrow().egui_ctx.clone();
```

Comment:
What is this `borrow()` that caused performance problems?

Reply:
Improvement : In glow and wgpu, we are removing egui_ctx and only using integration.egui_ctx.

Fix Issues: On Windows, we should use EventResult::RepaintNow() in Immediate Viewport to avoid FPS slowdown.